build.gradle

```
@@ -240,6 +240,7 @@ release {
     failOnPublishNeeded = false
 
     buildTasks = ['build']
+    failOnSnapshotDependencies = true
```

```suggestion
    failOnPublishNeeded = false
    failOnSnapshotDependencies = true
    
    buildTasks = ['build']
```

* I think this way it is much more readable.